dashboard/dashboard.py

```diff
@@ -0,0 +1,46 @@
+"""Script that creates the dashboard for the archiver data"""
+from os import environ
+import logging
+import re
+
+from dotenv import load_dotenv
+import pandas as pd
+import streamlit as st
+import altair as alt
+
+from extract import get_connection, get_all_data
+from dashboard_functions import make_hourly_archive_tracker_line, make_archive_searchbar, make_popular_archives_bar
+
+
+def make_url_alias(url):
+    regex_pattern = r'^(https?:\/\/)?((?:www\.)?)([^\/]+)'
```

Comment:
I know it's just a rough start, but maybe add docstrings for the functions soon-ish (just so you don't have to do it all at once at the end!)